tests/e2e/local/minikube/install_prereqs_debian.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-source "../common_linux.sh"
```

Comment:
OK, you didn't understand what I mean so I will write the code I expect for each file:

source "../common_linux.sh"